src/pages/index.js

```diff
@@ -1,94 +1,216 @@
+/* ------------------------- CREATE AND RENDER CARDS ------------------------ */
+function createCard(cardData) {
+  const cardElement = new Card(
+    cardData,
+    "#card-template",
+    handleLikeClick,
+    handleDeleteClick,
+    (link, name) => {
+      imagePreview.open(link, name);
+    }
+  );
+  return cardElement.getView();
+}
 
-const addCardPopup = new PopupWithForm(
-  "#profile-add-card-modal",
-  handleAddCardSubmit
-);
-addCardPopup.setEventListeners();
+function renderCard(cardData) {
+  const element = createCard(cardData);
+  cardSection.addItem(element);
+}
 
-const editProfilePopup = new PopupWithForm(
-  "#profile-edit-modal",
-  handleProfileEditSubmit
-);
-editProfilePopup.setEventListeners();
+/* ------------------------ USER INFO AND INITAL CARS ----------------------- */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "60774818-2b72-4a3b-8d83-5b2abf9c8a5e",
+    "Content-Type": "application/json",
+  },
+});
 
-const imagePreview = new PopupWithImage("#popup-modal");
-imagePreview.setEventListeners();
+api
+  .loadPageContent()
+  .then(([cards, userData]) => {
+    newUserInfo.setUserAvatar(userData.avatar);
+    newUserInfo.setUserInfo({
+      name: userData["name"],
+      about: userData["about"],
+      id: userData["_id"],
+    });
+    cardSection = new Section(
+      {
+        items: cards,
+        renderer: renderCard,
+      },
+      ".cards__list"
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
-const renderCard = (cardData) => {
-  const createCard = new Card(cardData, "#card-template", handleImageClick);
-  cardSection.addItem(createCard.getView());
-};
+let cardSection;
 
-const cardSection = new Section(
-  { items: initialCards, renderer: renderCard },
-  ".cards__list"
+const newUserInfo = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
 );
-cardSection.renderItems();
 
-const profileEditFormValidator = new FormValidator(config, profileEditForm);
-profileEditFormValidator.enableValidation();
-
-const addCardFormValidator = new FormValidator(config, addCardForm);
-addCardFormValidator.enableValidation();
-
-function handleImageClick(title, link) {
-  imagePreview.open(title, link);
-}
+/* ------------------------------ ADD NEW CARD ------------------------------ */
+const addCardPopup = new PopupWithForm("#add-card-modal", handleAddCardSubmit);
+addCardPopup.setEventListeners();
 
 function handleAddCardSubmit(data) {
-  renderCard(data);
-  addCardPopup.close();
+  addCardPopup.setLoading(true);
+  api
+    .addCard(data)
+    .then((res) => {
+      renderCard(res);
+      addCardPopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => addCardPopup.setLoading(false, "Create"));
 }
 
+addCardOpenButton.addEventListener("click", () => {
+  formValidators["edit-profile-form"].resetValidation();
```

Comment:
`"edit-profile-form"` is for the profile form. You need to use `add-card-form`

* You need to **`disable`** the submit button in the `card popup` if the inputs are **empty** when you open it. It’s needed because I can click the button and add an empty card but it should **not** happen. After the first submit the button gets activated and you need to **deactivate** it again. Even if I can't add an empty card the button should look **disabled** (gray) anyway and with attribute `disabled`.